From cc7a684862fee31c442c49d6a537d86979c23220 Mon Sep 17 00:00:00 2001
From: Thomas Deutschmann <whissi@gentoo.org>
Date: Thu, 17 Nov 2016 17:19:35 +0100
Subject: [PATCH] configure: Fix --with-audit parameter check

Due to a missing "$" the variable "$useaudit" was never checked for
0 value with the result that you cannot disable building against
libaudit.

Gentoo-Bug: https://bugs.gentoo.org/540446
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 7e7634f..708d1dd 100644
--- a/configure.in
+++ b/configure.in
@@ -822,7 +822,7 @@ AC_ARG_WITH(audit,
     ;;
   esac ]
 )
-if test useaudit = "0"; then
+if test "$useaudit" = 0; then
   AC_MSG_RESULT(no)
 elif test "$useaudit" = "1" && test "$auditavail" != 1; then
   useaudit=0
-- 
2.10.2

